tools/lkl/tests/net.sh

```diff
@@ -126,14 +122,16 @@ run_tests()
         lkl_test_exec $script_dir/net-test --backend pipe \
                       --ifname "$fifo1|$fifo2" \
                       --ip $(ip_host) --netmask-len $TEST_IP_NETMASK \
-                      --sleep 30 >/dev/null &
+                      --sleep 1800 >/dev/null &
         cp $script_dir/net-test $script_dir/net-test2
+        sleep 10
```

Comment:
Maybe ping the instance until it responds, to avoid a fixed sleep here?

Reply:
updated by using hijacked ping.  how 'bout this ?

Reply:
no, tests didn't pass unfortunately...
because i used hijack ping but it's not supported on FreeBSD.